### ISSUE TITLE
[Accessibility]A11y_PowershellEditorServices_Installation_Keyboard:Keyboard focus is not visible on "Settings","Commands","Languages","Color Themes","Debuggers" button.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#61713]
**Environment Details:**
VSCode Version : 1.25.1
OS Version : Win10

**Repro Steps:**
1.Launch Visual Studio code desktop application.
2.Navigate to "Extension" button of "ActivityBar"->Enter "PowerShell" in "Search Extension" text field and click on "Install" button.
3.Navigate through keyboard such as "Tab" key.
4.Observed that whether focus is visible on the "Settings","Commands","Languages","Color Themes","Debuggers" button under the Details Tab.

**Actual:**
Focus is not visible on "Setting","Command","Language","Color Themes","Debuggers" though it is actionable and NVDA is reading.

**Expected:**
Keyboard focus should visible on "Setting","Command","Language","Color Themes","Debuggers" button.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](url)

**User Impact:**
User will not able see the that where is the focus.

**MAS Reference:**
MAS2.4.7
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame2.aspx?sourcedoc=%7B3b98b04f-4fab-4d3e-81af-a0906a34849c%7D](url)
Does this issue occur when all extensions are disabled?: Yes
**Attachment for Reference:**
[2731192_A11y_PowershellEditorServices_Installation_Keyboard.pptx](https://github.com/Microsoft/vscode/files/2510442/2731192_A11y_PowershellEditorServices_Installation_Keyboard.pptx)
